File: IOSwiftUI/Backoffice.UI/src/screens/memberimages/models/BOImagesListResponseModel.ts
import BaseResponseModel from "../../../common/models/BaseResponseModel";
import BOImagesModel from "./BOImagesModel";

class BOImagesListResponseModel extends BaseResponseModel {

    count: number;
    imagesList: BOImagesModel[];

    constructor() {
        super();

        this.count = 0;
        this.imagesList = [];
    }
}

export default BOImagesListResponseModel;

File: IOSwiftUI/Common/Messages/Images/BOImagesListResponseModel.cs
using System;
using System.Collections.Generic;
using IOSwiftUI.Common.Messages.Base;

namespace IOSwiftUI.Common;

public class BOImagesListResponseModel : ResponseModel
{
    public int Count { get; set; }
    public IList<BOImagesModel> ImagesList { get; set; }

    public BOImagesListResponseModel(int count, IList<BOImagesModel> imagesList) : base()
    {
        Count = count;
        ImagesList = imagesList;
    }
}
